docs/content/Template.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Action Menu
+status: Alpha
+---
+
+<!-- Start with a Description about the `Component` and when would you use it -->
+
+An action menu is a vertical list of interactive actions. It’s composed of items presented in a consistent single-column format, with room for icons, descriptions, side information, and other rich visuals.
+
+<br />
+
+<!-- Preview up top, so that folks can figure out if this is the component they want -->
+<!-- Rendered version is ideal, but an image is a good fallback -->
+
+![Action menu preview](https://user-images.githubusercontent.com/293280/123880964-95b3a700-d8f8-11eb-9775-cbaf165207ed.png)
```

Comment:
I would love for this to be a live code example. Maybe something similar to the Radix Primitives examples: https://www.radix-ui.com/docs/primitives/components/accordion

Reply:
Yeeep! 

I struggled to create a `defaultOpen` action menu that doesn't interfere with scroll! I think I have new hacks now, I can give another try!